src/cli.ts

```diff
@@ -135,19 +142,24 @@ export function runTranspile(files: string[], options: CliOptions) {
 
 program
   .command('transform <file>')
+  .description(
+    'Debug tool which applies any set of passes to the AST and writes out the transformed Solidity',
+  )
   .option('--no-compile-errors')
-  .option('--check-trees')
-  .option('--highlight <ids...>')
-  .option('--order <passOrder>')
-  .option('-o, --output-dir <path>')
-  .option('--print-trees')
-  .option('--no-result')
```

Comment:
What about this option?

Reply:
On the right is the new ones with the added descriptions

Reply:
Yes, I just meant that `--no-result` option was removed, no description was provided for it either, so I wanted to know if this was something that was intended

Reply:
I think this was a mistake in https://github.com/NethermindEth/warp/pull/827/commits/1e94ea68bcf66036e2907ed313817f3323da2b98

Reply:
Correction, this option didn't do anything anymore. It has been removed in https://github.com/NethermindEth/warp/commit/1e94ea68bcf66036e2907ed313817f3323da2b98